src/gas-hub.js

```diff
@@ -398,6 +418,44 @@ function updateBranch() {
   });
 }
 
+function handleRepoCreated(repo) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/79a1121575f894385b70c00ba30d867da9fde2de/src/gas-hub.js#L421"></span>‘handleRepoCreated’ is defined but never used.</td>
    </tr>
  </tbody>
</table>